src/inference/include/openvino/runtime/auto/properties.hpp

```diff
@@ -29,6 +29,12 @@ static constexpr Property<bool> enable_startup_fallback{"ENABLE_STARTUP_FALLBACK
  */
 static constexpr Property<bool> enable_runtime_fallback{"ENABLE_RUNTIME_FALLBACK"};
 
+/**
+ * @brief auto device saves the hash id of the blob file for each device and its existence status
+ */
+static constexpr Property<std::map<std::string, std::pair<std::string, bool>>> devices_blob_hash_id{
```

Comment:
Can be some more description be added what is key and value of the this map.
Use unordered map, sorting probably has no meaning here.

Could provide more details how it works (the flow)
Why core cannot store computed hashes inside and AUTO can get them via internal property?